models/linkages/R/write.restart.LINKAGES.R

```diff
@@ -0,0 +1,234 @@
+write.restart.LINKAGES <- function(nens,outdir,run.id,time,settings,prior,analysis){
+  
+  biomass_function<-function(dbh){ #kg/tree
+      .1193 * dbh^2.393 + ((slta+sltb*dbh)/2)^2 * 3.14 * fwt * frt * .001
+  }
+  merit<-function(dbh){
+    (b_obs - biomass_function(dbh))^2
+  }
+  
+  distance.matrix <- rbind(c(0,3,1,2),
+                           c(3,0,2,1),
+                           c(1,2,0,3),
+                           c(2,1,3,0))
+  
+  PLOT = FALSE
+
+  ##HACK
+  spp.params.default <- read.csv(system.file("spp_matrix.csv", package = "linkages")) #default spp.params
+  nspec <- length(settings$pfts)
+  spp.params.save <- numeric(nspec)
+  for(i in 1:nspec){
+    spp.params.save[i] <- which(spp.params.default[,1]%in%settings$pfts[i]$pft$name)
+  }
+  
+  spp.params <- spp.params.default[spp.params.save,]
+  ##HACK
+  
+  analysis <- analysis[,c(1,2,4,3)] #doing this to match the linkages order #need to figure out how to generalize
+  
+  for(i in 1:nens){
+    # skip ensemble member if no file availible  
```

Comment:
equivalent question for write.restart.MODEL, whether the design is to be passed an ensemble or an individual run.